subsys/bluetooth/audio/mpl.c

```diff
@@ -0,0 +1,2923 @@
+/*  Media player skeleton implementation */
+
+/*
+ * Copyright (c) 2019 - 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Please tell me if this is an unreasonable request for this PR: Since the Media Player is not BT specific, and in fact probably shouldn't even reside in `zephyr/subsystem`, would it require much to move this to a sample instead, which it technically is?

Reply:
I would prefer to do that as a separate PR (if it is to be done).
But I am also not sure that would be wise - it would make the stack depend upon a sample, which seems the wrong way around.

Reply:
The stack doesn't really depend on the media player, does it? Sure, you can't use the media proxy without it, but it should still build just fine. 

As I see it at least, both the media player and the media controller could reside outside of the Zephyr Subsystem/Stack, but let's continue this outside the scope of this PR :)